### DISCUSSION TITLE
Badge or color change for determining articles from videos.

### DISCUSSION THREAD

**Discussion by jkasavage** [dailydotdev/daily#698] [Feature Requests]
I know its not the case for everyone but I prefer to learn through article research and practical experience. I absolutely despise video content unless it's super specific. Could we possibly have an indicator or something of the sort to differentiate between video content and article content? I checked to see if there were similiar ideas but I did not find anything concrete regarding a feature like this. Thank you and keep up the great work!!

**Comment by idoshamun** [comment]
Hi @jkasavage,

Thanks for the suggestion. Right now, we are very focused on textual content. That said, sometimes a video slips in through our system, so you shouldn't see them much. Once we add official support in videos, we will definitely take your feedback into account.
